app/models/course/assessment/question/programming_test_case.rb

```diff
@@ -6,4 +6,9 @@ class Course::Assessment::Question::ProgrammingTestCase < ActiveRecord::Base
 
   belongs_to :question, class_name: Course::Assessment::Question::Programming.name,
                         inverse_of: :test_cases
+  has_many :test_results,
+           class_name: Course::Assessment::Answer::ProgrammingAutoGradingTestResult.name,
+           inverse_of: :nil,
```

Comment:
As in, need to specify that it's `inverse_of: :test_cases` ? or something along those lines.